Firestore/Source/Remote/FSTSerializerBeta.mm

```diff
@@ -1192,7 +1193,7 @@ - (FSTDocumentWatchChange *)decodedDocumentRemove:(GCFSDocumentRemove *)change {
 
 - (FSTExistenceFilterWatchChange *)decodedExistenceFilterWatchChange:(GCFSExistenceFilter *)filter {
   // TODO(dimond): implement existence filter parsing
```

Comment:
I don't know if this TODO is still relevant.

Reply:
I suspect this is related to the above mentioned bug. I'm in favour of removing the TODO.

Reply:
Done (also removed another TODO in spec tests which also seemed to presume more kinds of existence filters: `extend this with different existence filters over time`).